src/main/java/org/example/calculation/ArithmeticCompute.java

```diff
@@ -0,0 +1,96 @@
+package org.example.calculation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+public class ArithmeticCompute implements Compute {
+    @Override
+    public double operate(String input){
+        return calculate(convertInput(input));
+    }
+
+    // 전위 표기식을 후위 표기식으로 변환.
+    public String[] convertInput(String input) {
+        String[] str = inputFormating(input);
+
+        List<String> sb = new ArrayList<>();
+        Stack<String> stack = new Stack<>();
+
+        for(int i = 0; i < str.length; i++){
+            String cur = str[i];
+
+            switch (cur){
+                case "+":
+                case "-":
+                case "*":
+                case "/":
+                    while (!stack.isEmpty() && priority(stack.peek()) >= priority(cur)){
+                        sb.add(stack.pop());
+                    }
+                    stack.push(cur);
+                    break;
+                default:
+                    sb.add(cur);
+            }
```

Comment:
```suggestion
            switch (cur) {
                    case "+" -> stack.push(targetNum2 + targetNum1);
                    case "-" -> stack.push(targetNum2 - targetNum1);
                    case "*" -> stack.push(targetNum2 * targetNum1);
                    case "/" -> stack.push(targetNum2 / targetNum1);
                }
```

와 같이 간결하게 사용할 수 있고, 추가적으로 https://openjdk.org/jeps/406 이글도 읽어보세요 ㅎㅎ